rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/KustomizeBakeManifestService.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ package com.netflix.spinnaker.rosco.manifests;
+
+ import com.netflix.spinnaker.kork.artifacts.model.Artifact;
+
+ import java.util.Map;
+
+ public interface KustomizeBakeManifestService {
```

Comment:
Shouldn't the interface be a `BakeManifestService` with  concrete implementations of `HelmBakeManifestService` and a `KustomizeBakeManifestService`?

Reply:
Looks like we can remove this class. `BakeManifestService` provides this interface.

Reply:
yeah, you right, removed.